.github/workflows/ci-frontend-a11y.yml

```diff
@@ -0,0 +1,62 @@
+name: pa11y tests
+
+on:
+  pull_request:
+    paths:
+      - frontend/**
+      - .github/workflows/ci-frontend-a11y.yml
+
+jobs:
+  build:
+    name: Pa11y-ci tests
+    runs-on: ubuntu-latest
```

Comment:
if we are running this locally, we may want to have it set to a matrix of OS' like the example pa11y tests show. Since locally we would be running on macOS-latest. 

name: test (${{ matrix.os }}, node ${{ matrix.node-version }})
    runs-on: ${{ matrix.os }}
    strategy:
      fail-fast: false
      matrix:
        os: [ubuntu-latest, windows-latest, macos-latest]
        node-version: [18, 20]